bodhi/server/models/models.py

```diff
@@ -2233,12 +2233,13 @@ def expire(self):
         if self.expired_date is not None:
             return
 
-        koji_session = buildsys.get_session()
-        try:
-            koji_session.untagBuild(self.build.release.override_tag,
-                                    self.build.nvr, strict=True)
-        except Exception, e:
-            log.error('Unable to untag override %s: %s' % (self.build.nvr, e))
+        if datetime.utcnow() >= self.expiration_date:
```

Comment:
Do you believe this method is getting called when it should not be? It might be cleaner to add this check to the first line of the method, to make it like this instead:

```
 if self.expired_date is not None or datetime.utcnow() < self.expiration_date:
```

This way the diff here is smaller and we get the same behavior as established above.